ingestion/tests/unit/topology/metadata/test_atlas.py

```diff
@@ -0,0 +1,612 @@
+                                description=None,
+                                fullyQualifiedName="delta.default.test_schema.delta_test_table.col2.col4",
+                                tags=None,
+                                constraint=None,
+                                ordinalPosition=None,
+                                jsonSchema=None,
+                                children=None,
+                                customMetrics=None,
+                                profile=None,
+                            ),
+                        ],
+                        customMetrics=None,
+                        profile=None,
+                    ),
+                    Column(
+                        name="col1",
+                        displayName=None,
+                        dataType="BIGINT",
+                        arrayDataType=None,
+                        dataLength=1,
+                        precision=None,
+                        scale=None,
+                        dataTypeDisplay="bigint",
+                        description="col1 description",
+                        fullyQualifiedName="delta.default.test_schema.delta_test_table.col1",
+                        tags=None,
+                        constraint=None,
+                        ordinalPosition=None,
+                        jsonSchema=None,
+                        children=None,
+                        customMetrics=None,
+                        profile=None,
+                    ),
+                ],
+                databaseSchema=EntityReference(
+                    id=mock_database_schema_object.id, type="databaseSchema"
+                ),
+            ),
+        )
+
+    def mock_ingest_lineage(self, source_guid, name):  # pylint: disable=unused-argument
+        return []
+
+    def mock_create_tag(self):
+        return []
+
+    @patch.object(AtlasClient, "list_entities", mock_list_entities)
+    @patch.object(AtlasClient, "get_entity", mock_get_entity)
+    @patch.object(AtlasSource, "ingest_lineage", mock_ingest_lineage)
+    @patch.object(AtlasSource, "create_tag", mock_create_tag)
+    def test_description(self):
+        """
+        Testing description updated for database, databaseSchema, table
+        """
+        _ = list(self.atlas_source.next_record())
+        updated_database = self.metadata.get_by_name(
+            entity=Database, fqn="hive.Reporting"
+        )
+        assert updated_database.description == EXPTECTED_DATABASE.description
+
+        updated_database_schema = self.metadata.get_by_name(
+            entity=DatabaseSchema, fqn="hive.Reporting.Reporting"
+        )
+        assert (
+            updated_database_schema.description == EXPTECTED_DATABASE_SCHEMA.description
+        )
+
+        updated_table = self.metadata.get_by_name(
+            entity=Table, fqn="hive.Reporting.Reporting.sales_fact_daily_mv"
+        )
+
+        assert updated_table.description == EXPTECTED_TABLE.description
```

Comment:
also apart from description we can also verify tags that we are updating for table